about.html

```diff
@@ -54,33 +54,41 @@ <h2 id="logo-title">Creative Commons Global Summit 2015 Logo</h2>
                                             process, the logo from Naresh Agrawal won the prize.
                                             After consultation with Mr. Agrawal we optimized the prize-winning-logo
                                             for this year´s summit as following:</p>
-        <div id="logo-about"><a href="index_home-html" alt=""><img src="images/logo-summit.jpg" alt=""></a></div>
+        <div id="logo-about"><a href="index.html" alt=""><img src="images/logo-summit.jpg" alt=""></a></div>
         
   
-        <h2 id="featured-title-past">See the past CC Global Summits</h2>
+        <h2 id="featured-title-past">See the past GM Art Expo</h2>
         <div class="underline past-desktop"></div>
-        <p class="about-intro past-intro">Take a look at the last two CC Global Summits which took place
-                                            in Buenos Aires and in Warsaw.</p>
+        <p class="about-intro past-intro">Take a look at the last two GM Art Expo which took place
+                                            in Miami and in New York.</p>
         <div id="past-images">
-          <div id="past-image1"><a href="#" alt=""><img src="images/2013global.jpg" alt=""></a></div>
-          <div id="past-image2"><a href="#" alt=""><img src="images/2011global.jpg" alt=""></a></div>
+          <div id="past-image1">
+            <img src="images/expo1.jpg" alt="">
+            <div class="img-textyear">2021</div>
+            <div class="img-text">GM Art Expo in Miami</div>
+          </div>   
+          <div id="past-image2">
+            <img src="images/expo2.jpeg" alt="">
+            <div class="img-textyear">2022</div>
+            <div class="img-text">GM Art Expo in NYC</div>
+          </div>
```

Comment:
- [ ] Kindly add the orange layer on the image here to match the provided design.

**your design**
![image](https://user-images.githubusercontent.com/74411135/235277942-ca9185a2-3a99-410a-8d14-6d382c3fb67b.png)

**provided design**
![image](https://user-images.githubusercontent.com/74411135/235278017-8d42940c-1140-4795-846a-d3ba797fc14e.png)

This will make your project prettier and make the user know you're an awesome developer as you can precisely follow a professional web developer's design 😃 💪